eng/native/configurecompiler.cmake

```diff
@@ -85,6 +85,14 @@ if (MSVC)
     set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /sourcelink:${CLR_SOURCELINK_FILE_PATH}")
   endif(EXISTS ${CLR_SOURCELINK_FILE_PATH})
 
+  if (CMAKE_GENERATOR MATCHES "^Visual Studio.*$")
+    # Debug build specific flags
+    # until https://github.com/dotnet/runtime/issues/64082 is fixed.
+    add_linker_flag(/INCREMENTAL:NO DEBUG)
+    add_linker_flag(/OPT:REF DEBUG)
+    add_linker_flag(/OPT:NOICF DEBUG)
+  endif (CMAKE_GENERATOR MATCHES "(Visual Studio)")
```

Comment:
```suggestion
  endif (CMAKE_GENERATOR MATCHES "^Visual Studio.*$")
```

Reply:
This is already fixed in [664302c](https://github.com/dotnet/runtime/pull/86170/commits/664302c3da5215ff0b8a6006bd03a16c92097a6b).

Reply:
That was the opening `if`, now Cmake warns about a mismatch because the closing `endif` has a different expression: https://dev.azure.com/dnceng-public/public/_build/results?buildId=272614&view=logs&j=39a07da7-60a1-56aa-621d-b74991bb2f3d&t=b6fb07db-b0d1-5e3d-34de-5cdd549faa27&l=41